### PR TITLE
chore(deploy-web): update Google Analytics configuration 

### PR DESCRIPTION
**Pull request by ghost**
add ga config to track flow from web to console

### CODE REVIEW

File: apps/deploy-web/src/components/layout/CustomGoogleAnalytics.tsx [hunk 1]
```diff
@@ -14,5 +14,14 @@ export default function GoogleAnalytics() {
     });
   });
 
-  return <>{isProd && <GAnalytics trackPageViews />}</>;
+  return (
+    <>
+      {isProd && (
+        <>
+          <GAnalytics trackPageViews />
+          <GAnalytics trackPageViews gaMeasurementId="G-LFRGN2J2RV" />
```
Comment (baktun14) [inline apps/deploy-web/src/components/layout/CustomGoogleAnalytics.tsx:22]: Which one is this? The main website? We should get this from an env variable

Comment (HoomanDigital) [reply apps/deploy-web/src/components/layout/CustomGoogleAnalytics.tsx:22]: yes from main website, we have to use same ga on both: Akash website and Akash console.. we can also keep it without env in this case..
WhatsApp Image 2024-07-24 at 8 57 47 PM

Comment (baktun14) [reply apps/deploy-web/src/components/layout/CustomGoogleAnalytics.tsx:22]: Ok but there's going to be 2 GA scripts?

Comment (HoomanDigital) [reply apps/deploy-web/src/components/layout/CustomGoogleAnalytics.tsx:22]: Yes, for the time being. We will remove the existing one once the second one starts functioning well as per our needs.

**Review by Redm4x — APPROVED** [review]

**Review by baktun14 — APPROVED** [review]

**Pull request merged**